app/main.py

```
@@ -324,7 +324,47 @@ async def download_event_ics(event_id: str, client: httpx.AsyncClient = Depends(
+    }
+
+    copyright_json, _ = await _cms_get(f"/copyrights", params, client)
+    copyright_info = copyright_json.get("data")
+    if not isinstance(copyright_info, list):
+        raise HTTPException(status_code=404, detail="Copyright not found")
+
+    return copyright_info
+
+
+@app.get("/copyright/{image_id}", description="Get copyright for one image", tags=["copyright"])
+async def get_one_copyright_info(image_id: str, client: httpx.AsyncClient = Depends(_get_cms_client)):
```

Ruff (0.15.12)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)